specification/common-types/resource-management/v3/managedidentity.json

```diff
@@ -80,31 +108,48 @@
       ],
       "x-ms-enum": {
         "name": "SystemAssignedServiceIdentityType",
-        "modelAsString": true
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "None",
+            "value": "None",
+            "description": "No managed system identity."
+          },
+          {
+            "name": "SystemAssigned",
+            "value": "SystemAssigned",
+            "description": "System assigned managed system identity."
+          }
+        ]
       }
     },
     "UserAssignedIdentities": {
       "type": "object",
-      "title": "User-Assigned Identities",
       "description": "The set of user assigned identities associated with the resource. The userAssignedIdentities dictionary keys will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}. The dictionary values can be empty objects ({}) in requests.",
       "additionalProperties": {
```

Comment:
I need to figure out the solution for this issue first.https://github.com/Azure/typespec-azure/issues/1163

Reply:
I am working on that

Reply:
this has been fixed pan!